oap-server/server-receiver-plugin/aws-firehose-receiver/src/main/proto/opentelemetry/proto/common/v1/common.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.common.v1;
```

Comment:
I think we should name package name as v0_7 suffix? This could be more clear. WDYT?

Reply:
The package is a relative concept, I think it's necessary.

Reply:
If we change package, the directory have to change as well

Reply:
I think it is fine to change. No one has used this.
I want to change because I feel one day, AWS would really pick up v1 or upgrade to a specific version.
Then we could be clear for changing from version to version.

Reply:
OK, `opentelemetry/v0_7/proto/.....` is better.

![image](https://user-images.githubusercontent.com/3917424/216808566-89cba900-2475-4cb3-a82d-d85271e2df39.png)

Reply:
Why do we keep v1?

Reply:
Adding a parent directory is simpler than adding a directory for each module. 
In the future, we could create a  directory named `v1`, and copy the whole OTEL proto files to the directory, rather than change each directory in OTEL

Reply:
We don't need to update folder structure, I just make sure Java package including correct version.